src/button.tsx

```diff
@@ -383,6 +383,12 @@ const Button: React.FC<ButtonProps & {classes: ReturnType<typeof usePrimaryButto
         role: 'button',
     };
 
```

Comment:
unrelated changes to make button more resilient, requested by @Winde

Reply:
we could consider using some `invariant` [library](https://www.npmjs.com/package/tiny-invariant) for this kind of checks. We have these checks in other components too. TS/Flow understand `invariant` calls and refine the types too.